packages/components/checkbox/src/checkbox.ts

```diff
@@ -121,7 +121,7 @@ export class Checkbox extends FormControlMixin(HintMixin(LitElement)) {
           <slot @slotchange=${() => this.#updateNoLabel()}></slot>
         </span>
       </div>
-      ${this.#validation.render()} ${this.#validation.render() ? this.#validation.render() : this.renderHint()}
+      ${this.#validation.render() ? this.#validation.render() : this.renderHint()}
```

Comment:
This looks weird; you're effectively rendering the validation *twice*? (the old code also looks weird)

Reply:
right... I need to find a better solution

Reply:
I've added some changes